src/Field/Field.tsx

```diff
@@ -32,28 +33,45 @@ export const Field = ({
   required,
   showCaret,
   fullHeight = false,
+  assistiveText,
   ...marginProps
 }: FieldProps) => {
   return (
     <Container className={className} {...marginProps}>
       {label && (
-        <Box mb={{ custom: outlined ? 4 : 0 }}>
+        <>
           {renderAsTitle ? (
-            <Text tag="label" typo="heading-small" htmlFor={id} mb="16px">
-              {label}
-            </Text>
-          ) : (
-            <Text tag="label" typo="label" color="subtext" htmlFor={id}>
-              {label}
-              {required && (
-                <Text tag="span" typo="body-small" color="error">
-                  *
+            <Box mb="16px">
+              <Text
+                tag="label"
+                typo="heading-small"
+                htmlFor={id}
+                mb={{ custom: 4 }}
```

Comment:
should this be `4` only if we have `assistiveText`? cause we already have the whole wrapper with `mb="16px"` so at that point we should probably put a `mt` to `assistiveText` instead?

Reply:
Yup goof point, fixed in [482a4](https://github.com/marshmallow-insurance/smores-react/pull/2118/commits/482a475fa27e58e0c51bd62be2c09b971e3c8f78)